.travis.yml

```diff
@@ -164,6 +193,8 @@ before_install:
       echo 'autotools is installed by default'
     elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
       brew install automake
+    elif [[ "$TRAVIS_OS_NAME" == "windows" ]]; then
+      $msys2 pacman --sync --noconfirm --needed automake-wrapper
```

Comment:
I think you should move `export MAKE=mingw32-make` down to here, if the comment is correct that it's only ever used by autotools and not by any of the other build systems.

Reply:
Ah, you're right. Come to think of it, I didn't use `mingw32-make` locally at all, even with `autotools`. That also reminds me that usual `make` needs to be installed.